app/guid-file/route.ts

```diff
@@ -33,6 +38,21 @@ export default class FileDetail extends Route.extend() {
         }
     }
 
+    async afterModel(this: FileDetail, model) {
+        const dateCreated = model.file.get('dateCreated');
+        const dateModified = model.file.get('dateModified');
+        const institutions = await model.file.get('user').get('institutions');
```

Comment:
Do we want to block rendering the page until all institutions are loaded? Could this happen in a task where it's actually asynchronous?

Reply:
Yeah, that would probably be better, but we have to make sure the head tags are defined before, say, the Facebook link sharer, parses the page. Are we handling this with prerender? In that case, I could make setting the head tags a task and set a ready blocker.

Reply:
Yeah, that seems like the best approach 👍